dnf-json

```diff
@@ -182,6 +216,22 @@ class Solver():
 
 class DnfJsonRequestHandler(BaseHTTPRequestHandler):
 
+    cache_folders = []
+
+    def init_cache_folder_list(self, repos):
+        for repo in repos:
```

Comment:
list comprehension and a nested convert function from `repo` to `f"{repoid}-{digest}" might be slightly more pythonic I think.

Reply:
Okay, will do.